packages/brand-shape/README.md

```diff
@@ -0,0 +1,28 @@
+# Blob
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/blob.svg)
+
+#### [View on MongoDB.design](https://www.mongodb.design/component/blob/example/)
+
+## Installation
+
+### Yarn
+
+```shell
+yarn add @leafygreen-ui/blob
+```
+
+### NPM
+
+```shell
+npm install @leafygreen-ui/blob
+```
+
+## Example
+
+**Output HTML**
+
+## Properties
+
+| Prop | Type | Description | Default |
+| ---- | ---- | ----------- | ------- |
```

Comment:
Need prop table, and some documentation on how the `o` vs. `O` in the `shape` prop should be expected to behave. 

There are also a few shapes that feel like they should work, but maybe they're not possible due to technical constraints - like:
```
<Blob
shape={[
["o","o","o","o"],
["o"," "," ","o"],
["o"," "," ","o"],
["o","o","o","o"]
]}
/>
```